### PR TITLE
add createElementNS

### PR DESCRIPTION
**Pull request by planetis-m**
https://developer.mozilla.org/en-US/docs/Web/API/Document/createElementNS No idea about nodejs

### CODE REVIEW

**Comment by planetis-m** [comment]
Ref: https://github.com/pragmagic/karax/pull/179

File: lib/js/dom.nim [hunk 1]
```diff
@@ -1467,6 +1467,7 @@ else:
   proc insertBefore*(n, newNode, before: Node) {.importcpp.}
   proc getElementById*(d: Document, id: cstring): Element {.importcpp.}
   proc createElement*(d: Document, identifier: cstring): Element {.importcpp.}
+  proc createElementNS*(d: Document, namespaceURI, qualifiedIdentifier: cstring): Element {.importcpp.}
```
Comment (timotheecour) [inline lib/js/dom.nim:1470]:
```suggestion
  proc createElementNS*(d: Document, namespaceURI, qualifiedIdentifier: cstring): Element {.importcpp.}
    ## https://developer.mozilla.org/en-US/docs/Web/API/Document/createElementNS
```

we should start adding doc refs everywhere IMO; it's more user friendly (and new js modules do that)

Comment (planetis-m) [reply lib/js/dom.nim:1470]: Ideally someone should look into https://github.com/nim-lang/Nim/issues/11021

**Review by timotheecour — APPROVED** [review]
LGTM , with or without the suggested change

**Pull request merged**